### PR TITLE
Integration with current FastRTPS security logging

### PR DESCRIPTION
**Pull request by artivis**

### CODE REVIEW

File: rmw_fastrtps_shared_cpp/src/rmw_node.cpp [hunk 1]
```diff
@@ -350,6 +350,7 @@ __rmw_create_node(
             security_logging_file_path,
             property_policy.properties()))
         {
+          RMW_SET_ERROR_MSG("couldn't parse security logging config file!");
```
Comment (kyrofa) [inline rmw_fastrtps_shared_cpp/src/rmw_node.cpp]: This is left off on purpose. RMW can only have one error message. By setting it here you overwrite the more informative messages already set that caused this function to return `false` in the first place.

Comment (artivis) [reply rmw_fastrtps_shared_cpp/src/rmw_node.cpp]: Oh ok my bad. Gonna remove the commit.

File: rmw_fastrtps_shared_cpp/src/rmw_security_logging.cpp [hunk 2]
```diff
@@ -111,8 +113,14 @@ bool apply_logging_configuration_from_file(
   const std::string & xml_file_path,
   eprosima::fastrtps::rtps::PropertySeq & properties)
 {
+  const auto stripped_xml_file_path =
+      std::regex_replace(xml_file_path, std::regex("file://"), "");
```
Comment (kyrofa) [inline rmw_fastrtps_shared_cpp/src/rmw_security_logging.cpp]: None of this should be necessary anymore. The underlying code has been refactored a bit to be more testable and this is no longer an issue.

**Comment by kyrofa** [comment]
The underlying code has been refactored to be more testable, which got rid of all the issues fixed here. Double check, but I no longer think this PR is necessary.

**Pull request closed without merging**